source/fundamentals/crud/sort.txt

```diff
@@ -0,0 +1,109 @@
+============
+Sort Results
+============
+
+.. default-domain:: mongodb
+
+Use ``sort()`` to change the order in which read operations return
+documents. The ``sort()`` instance method tells MongoDB to order returned documents by the
+values of one or more fields in a certain direction. To sort returned
+documents by a field in ascending (lowest first) order, use the static method ``Sorts.ascending()``, 
+specifying the field on which to sort. To sort in descending (greatest first) order instead, 
+use the static method ``Sorts.descending()``, specifying the field on which to sort. For example,
+you can sort on ``id`` or ``name`` fields. If you do not specify a sort, MongoDB does not 
+guarantee the order of query results.
+
+Follow the instructions in the examples below to insert data into
+a collection and perform a sort on the results of a query.
+Consider a collection containing documents that describe books. To
+insert this data into a collection, run the following operation:
+
+.. code-block:: java
+
+    collection.insertMany(Arrays.asList(
+        new Document().append("_id", 1)
+            .append("title", "The Brothers Karamazov").append("length", 824)
+            .append("author", "Dostoyevsky"),
+        new Document().append("_id", 2)
+            .append("title", "Les Misérables").append("length", 1462).append("author", "Hugo"),
+        new Document().append("_id", 3)
+            .append("title", "Atlas Shrugged").append("length", 1088).append("author", "Rand"),
+        new Document().append("_id", 4)
+            .append("title", "Infinite Jest").append("length", 1104).append("author", "Wallace"),
+        new Document().append("_id", 5)
+            .append("title", "Cryptonomicon").append("length", 918).append("author", "Stephenson"),
+        new Document().append("_id", 6)
+            .append("title", "A Dance with Dragons").append("length", 1104)
+            .append("author", "Martin")
+    ));
+
+Call the ``sort()`` function and specify the order with ``ascending()`` to ensure that the
+operation returns books in alphabetical order by title:
+
+.. code-block:: java
+  :emphasize-lines: 5
+
+    import com.mongodb.client.model.Sorts;
```

Comment:
I don't think this is enough to make `ascending` work below.  You either have to [statically import](https://docs.oracle.com/javase/8/docs/technotes/guides/language/static-import.html) `Sorts.ascending` or use `Sorts.ascending` in the code.